### ISSUE TITLE
[Terminal > Export Text]: Keyboard only users do not have option to use 'Export text' functionality.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12052]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Verify that keyboard only users can Export text or not.

**User Experience:**
Keyboard only users UX will not be good as they are not able to use a certain functionality.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

**Attachments:**
[Keyboard only users do not have option to use 'Export text' functionality..zip](https://github.com/microsoft/terminal/files/7782917/Keyboard.only.users.do.not.have.option.to.use.Export.text.functionality.zip)

### Expected Behavior

Keyboard only users should be able to use 'Export text' functionality.

### Actual Behavior

Keyboard only users does not have option to use 'Export text' functionality.

**Comment by zadjii-msft** [comment]
@v-rpundir would it be sufficient to have an `exportBuffer` action which the user can bind to a keybinding for exporting the text? (As discussed in #9700, #11090)? They would still have access to the functionality, but via the command palette / keybindings instead of via the tab context menu.

**Comment by v-rpundir** [comment]
@zadjii-msft Yup exportBuffer action will be sufficient for exporting the text.

**Comment by carlos-zamora** [comment]
@v-rpundir  This is now fixed in v1.13.10333.0 (Terminal Preview). Can you please confirm?

**Comment by v-rpundir** [comment]
Hi @carlos-zamora the latest build I have is Terminal Preview 1.12.3472.0 so I will keep this work in my To-Do list for now and will check when mentioned build is available here.

It will be better if you can provide us mentioned build us to verify the same.

**Comment by zadjii-msft** [comment]
@v-rpundir You can manually install the latest build using the `.msixbundle` from our Releases page here: https://github.com/microsoft/terminal/releases/tag/v1.13.10336.0

**Comment by v-rpundir** [comment]
Closing as Export Actions is available now on Windows Terminal Preview Version: 1.13.10336.0
